cmd/tink-cli/cmd/root.go

```diff
@@ -15,6 +15,7 @@ var rootCmd = &cobra.Command{
 	Use:               "tink",
 	Short:             "tinkerbell CLI",
 	PersistentPreRunE: setupClient,
+	DisableAutoGenTag: true,
```

Comment:
This prevents the markdown docs from create datestamp jitter, should we include and require updated docs/ in each commit.